### PR TITLE
Harvest attach

### PR DESCRIPTION
**Pull request by noirbizarre**
This pull-request add the `udata harvest attach` command allowing to link existing datasets to their remote ID from a CSV before harvest.

It also fixes the ODS backend HTML to Markdown conversion

### CODE REVIEW

File: udata/harvest/actions.py [hunk 1]
```diff
@@ -162,3 +164,43 @@ def unschedule(ident):
...
+    The expected csv file format is the following:
+
+    - a column with header "local" and the local IDs or slugs
+    - a column with header "remote" and the remote IDs
+
+    The delimiter should be ";". columns order
+    and extras columns does not matter
+    '''
+    count = 0
+    with open(filename) as csvfile:
```
Comment (davidbgk) [inline udata/harvest/actions.py]: No potential encoding issue?

File: udata/harvest/actions.py [hunk 2]
```diff
@@ -162,3 +164,43 @@ def unschedule(ident):
...
+        reader = csv.DictReader(csvfile,
+                                delimiter=b';',
+                                quotechar=b'"')
+        for row in reader:
+            try:
+                dataset = Dataset.objects.get(id=ObjectId(row['local']))
+            except:  # noqa  (Never stop on failure)
```
Comment (davidbgk) [inline udata/harvest/actions.py]: Might be interesting to have a counter for failures too? (I don't know if it happens often)

**Comment by davidbgk** [comment]
r+wc

File: udata/harvest/actions.py (hunk 1 above)
Comment (noirbizarre) [inline udata/harvest/actions.py]: It's only IDs, no unicode but your right, maybe I should use unicodecsv here too.

File: udata/harvest/actions.py (hunk 2 above)
Comment (noirbizarre) [inline udata/harvest/actions.py]: Errors are logged into the shell but I can also return a tuple `(success, errors)`

**Pull request merged**